server/integration/infinispan/src/main/resources/subsystem-templates/infinispan-core.xml

```diff
@@ -29,27 +23,27 @@
             <transport lock-timeout="60000" />
             <global-state />
 
-            <distributed-cache-configuration name="transactional" mode="SYNC" start="EAGER">
+            <distributed-cache-configuration name="transactional" mode="SYNC">
```

Comment:
@tristantarrant Is it worth making the mode attribute optional in order to further simplify the config? With a default value of "SYNC" if no mode is specified.